src/libraries/System.Net.NameResolution/src/System/Net/Dns.cs

```diff
@@ -16,20 +16,20 @@ public static class Dns
         /// <summary>Gets the host name of the local machine.</summary>
         public static string GetHostName()
         {
-            long startingTimestamp = NameResolutionTelemetry.Log.BeforeResolution(string.Empty);
+            NameResolutionActivity activity = NameResolutionTelemetry.Log.BeforeResolution(string.Empty);
```

Comment:
If I'm reading this right, turning on any diagnostics will also turn on the experimental Activity creation. Is that what we want?